modules/alloy/src/main/ts/ephox/alloy/positioning/view/Transitions.ts

```diff
@@ -42,7 +42,7 @@ const hasChanges = (position: PositionCss, intermediate: Record<TransitionProp,
 
 const getTransitionDuration = (element: SugarElement<HTMLElement>): number => {
   const get = (name: string) => {
-    // Some older browsers return `null` here (e.g. PhantomJS)
+    // Some older browsers return `null` here (so we probably don't need the string check anymore?)
```

Comment:
should there be a todo on this or something? is it something we actually want to follow up on?

Reply:
I ended up implementing the todo, deleted it at Lee's suggestion :)